packages/host/app/components/matrix/room-message.gts

```diff
@@ -230,6 +242,9 @@ export default class RoomMessage extends Component<Signature> {
         min-height: 7rem;
         max-height: 30vh;
       }
+      .embedded-message-field {
+        --boxel-field-embedded-padding: 0;
```

Comment:
Im certain someone has comment on this. Essentially, embedded delegated render is offering a padding by default. I added this to remove it but this would also mean any nested embedded inside this embedded will also have 0 padding